src/axom/primal/geometry/Polygon.hpp

```diff
@@ -101,6 +101,86 @@ class Polygon
     return sum;
   }
 
+  /**
+   * \brief Returns the area of the polygon (3D specialization)
+   *
+   * The algorithm sums up contributions from triangles defined by each edge 
+   * of the polygon and a local origin (at the Polygon's \a vertexMean()).
+   * The algorithm assumes a planar polygon embedded in 3D, however it will return 
+   * a value for non-planar polygons.
+   *
+   * The area is always non-negative since 3D polygons do not have a unique orientation.
+   */
+  template <int TDIM = NDIMS>
+  typename std::enable_if<TDIM == 3, double>::type area() const
```

Comment:
New functions in support of testing `Polyhedron::volume()` for rotated cones with a polygonal base. (See updated `primal_polyhedron` tests).